lib/cspScripts.ts

```diff
@@ -45,7 +46,7 @@ export const generateCSP = (): { csp: string; nonce: string } => {
   const cspHeader = `
     default-src 'self';
     script-src 'self' 'nonce-${nonce}' 'strict-dynamic';
-    style-src 'self' 'nonce-${nonce}';
+    style-src 'self' ${styleSrc};
```

Comment:
Curious why the change in value to a new declared variable?
```suggestion
    style-src 'self' 'nonce-${nonce}'
```